doobie/src/main/scala/za/co/absa/fadb/doobiedb/package.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.fadb
+
+import doobie.util.meta.Meta
+import doobie.implicits.javasql._
+
+import java.sql.Timestamp
+import java.time.ZoneId
+
+// can't be named doobie due to a naming conflict
+package object doobiedb {
+
+  object postgres {
```

Comment:
please verify that this is only related to PG. If not, then this library is even more universal :)

Reply:
hopefully will get to it soon

Reply:
After downgrade of Doobie version this is no longer relevant. Package object was removed as it became obsolete.